install.sh

```diff
@@ -2,54 +2,56 @@
 
 # Function to check for macOS PrusaSlicer directories
 mac_directories() {
-    directories=()
     if [ -d "$HOME/Library/Application Support/PrusaSlicer" ]; then
-        directories+=("$HOME/Library/Application Support/PrusaSlicer")
+        echo "$HOME/Library/Application Support/PrusaSlicer"
     fi
     if [ -d "$HOME/Library/Application Support/PrusaSlicer-alpha" ]; then
-        directories+=("$HOME/Library/Application Support/PrusaSlicer-alpha")
+        echo "$HOME/Library/Application Support/PrusaSlicer-alpha"
     fi
     if [ -d "$HOME/Library/Application Support/PrusaSlicer-beta" ]; then
-        directories+=("$HOME/Library/Application Support/PrusaSlicer-beta")
+        echo "$HOME/Library/Application Support/PrusaSlicer-beta"
     fi
```

Comment:
@spuder Please add
`$HOME/Library/Application Support/AnkerMake_alpha-alpha`
and yes alpha is in there tiwce.

Reply:
Since I'm not the original author, it will be easier for me to add this in a new PR (instead of forking a fork)

```
    if [ -d "$HOME/Library/Application Support/AnkerMake_alpha-alpha" ]; then
        echo "$HOME/Library/Application Support/AnkerMake_alpha-alpha"
    ]
```